### PR TITLE
325 enhancement add heroku config

### LINKED ISSUE #325: Enhancement: Add Heroku Config
**Describe the Current Behavior/Feature**:
Currently, we have a config for deploying to Github Pages, and a heroku instance.

**Proposed Behavior/Feature**:
We don't need the Github Pages deploy any more, and we need to make an update to NPM scripts to correctly launch the app on Heroku.

**Proposed Implementation (if applicable)**:
We can remove the Github Pages related actions from github actions, and we need to add an NPM start script. That's the script heroku uses to launch the app.

### PR DESCRIPTION
**Pull request by timbot1789**
Add config to run Pass from Heroku. Heroku itself is already set up to watch our repository, and is watching master for pushes. Heroku will see the next push to master, and redeploy our code. As long as these changes are in the push to master, the app should launch fine.

Usually, Heroku will be running master. But right now it is running this PR. It will automatically start running master again after then next push. You can see it here: https://passsmartwallet-967e217a2652.herokuapp.com/

Based these changes off the instructions in this stack overflow post: https://stackoverflow.com/questions/69444225/how-do-i-deploy-to-heroku-using-vite

Basically, Heroku runs the vite build process for us, then we serve the app with NGINX.

### CODE REVIEW

**Review by leekahung — APPROVED** [review]
Thanks Tim for setting up the app with Heroku! Just checked out the application site on my phone. It seems to be running fine. I approve.
